docs/src/formulation-details.md

```diff
@@ -0,0 +1,85 @@
+# Formulation Details
+
```

Comment:
We should add this reference, Daniel Molzahn and Ian Hiskens. ”A Survey of Relaxations and Approximations of the Power Flow Equations" Foundations and Trends in Electric Energy Systems, forthcoming.

Reply:
done!